execution/sym_region_frag_machine.mli

```diff
@@ -26,7 +26,7 @@ sig
     method get_eip : int64
     method set_eip : int64 -> unit
 
-    method eval_addr_exp_region : Vine.exp -> int -> (int option * int64)
+    method eval_addr_exp_region : Vine.exp -> int -> (int option * int64 * bool * Vine.exp * int64)
```

Comment:
Now that we see the interface of this method changing and becoming more complicated, I think it might make more sense to make it private rather than keeping it as part of the SRFM MLI interface. If there was any other code that used this method, it would be broken by this interface change, but it looks like there isn't, and it seems internal enough that it's not so likely to be useful from elsewhere in the future. I left a lot of things public by default when I first modularized this code, but I think going in the direction of fewer things being public will help maintainability in the future.